src/zod/accommodation-create.schema.ts

```diff
@@ -0,0 +1,39 @@
+import { z, type ZodType } from 'zod';
+import { type AccommodationFormData } from '#/modules/home/types/accommodation-form-data.interface';
```

Comment:
what do you think of using inferred types from the schema instead of this.

```js
type bar = z.infer<typeof CreateAccomodationSchema>
```

Reply:
fixed images and types